currency-rates/internal/services/mail.go

```diff
@@ -1,42 +1,87 @@
 package services
 
 import (
+	"crypto/tls"
 	"net/smtp"
 
 	"go.uber.org/zap"
 )
 
+const serverName = "smtp.gmail.com"
 type Mail struct {
 	from string
 	pass string
+	client   *smtp.Client
 	l *zap.SugaredLogger
 }
 
-func NewEmail(from string, pass string, l *zap.SugaredLogger) *Mail {
+func NewEmail(from string, pass string, l *zap.SugaredLogger) (*Mail, error) {
+	client, err := smtp.Dial("smtp.gmail.com:587")
+	if err != nil {
+		return nil, err
+	}
+
+	tlsconfig := &tls.Config{
+		ServerName: serverName,
+	}
```

Comment:
There is no need to create variable that is used only once.

Reply:
Done